api/src/services/stardust/influx/influxQueries.ts

```diff
@@ -335,4 +335,15 @@ export const MILESTONE_STATS_QUERY = `
         last("no_payload_count") AS "noPayload"
     FROM "stardust_block_activity"
 `;
+export const MILESTONE_STATS_QUERY_BY_INDEX = `
+    SELECT
+        last("milestone_index") AS "milestoneIndex",
```

Comment:
I don't think the `last` part makes sense here. `last` is from the other query where we specifically need the "latest row" in the database (it's sorted by time by default in influx). Here we are looking for a specific milestone, maybe the query works but i'm concerned that it might not return the right data.
I would remove the `last` and just check the query works without it.